sklearn/linear_model/_coordinate_descent.py

```diff
@@ -1459,6 +1459,29 @@ def _path_residuals(
 class LinearModelCV(MultiOutputMixin, LinearModel, ABC):
     """Base class for iterative model fitting along a regularization path."""
 
+    _parameter_constraints = {
+        "eps": [Interval(Real, 0, None, closed="left")],
+        "n_alphas": [Interval(Integral, 0, None, closed="left")],
+        "alphas": [np.ndarray, None],
+        "fit_intercept": [bool],
```

Comment:
We have a `"boolean"` helper constraint which also allows for things like `np.bool_`.
```suggestion
        "fit_intercept": ["boolean"],
```